docs/contributing/index.md

```diff
@@ -0,0 +1,24 @@
+# Contributing to Projects with GitHub for Visual Studio
+
+Use GitHub for Visual Studio to manage your projects and work with Pull Requests.
+
+### Table of Contents
+
+- Adding and cloning repositories
+  - [Publishing an existing project to GitHub](publishing-an-existing-project-to-github.md)
+  - [Creating an empty repository from Visual Studio](creating-an-empty-repository-from-visual-studio.md)
+  - [Cloning a repository to Visual Studio](cloning-a-repository-to-visual-studio.md)
+- Working with Pull Requests
+  - [View the Pull Requests for a repository](view-the-pull-requests-for-a-repository.md)
```

Comment:
> View the Pull Requests for a repository

Your other procedurals are titled with gerunds (which is consistent with the GitHub Help) - can this be "Viewing pull requests for a repository" to make it parallel?

To be consistent with GitHub Help, I recommend not capitalizing "issues" or "pull requests" in general (https://manual-of-style.githubapp.com/capitalization/#product-features):

> *A note on feature capitalization: Use your best judgment when using title case to refer to product features. For example, use sentence case if capitalizing a feature in the middle of the sentence looks jarring or inconsistent.*